src/internal/rest.rs

```diff
@@ -186,6 +184,8 @@ pub struct SignatureUrlString(String);
 impl SignatureUrlString {
     pub fn new(encoded_full_url: &str) -> Result<SignatureUrlString, url::ParseError> {
         let parsed_url = Url::parse(encoded_full_url)?;
+        //The formatter for the query string is overfomatting compared to what we want, so we want to replace
```

Comment:
Do you think this is worth opening an issue on reqwest? Something like "Support configurable query string encoding". Perhaps if we reference our issue someone will point out a better way all together?